packages/melos/lib/src/common/workspace.dart

```diff
@@ -246,6 +246,18 @@ class MelosWorkspace {
       packagesNoScope = packages;
     }
 
+    // --flutter / --no-flutter
+    if (hasFlutter != null) {
+      var pool = Pool(10);
```

Comment:
A pool isn't required on this one, guessing you copied pooling logic from the above 'since' flag, that one is a bit more intense and has calls to git cli hence why it's pooled.

But for this one you should be able to do a simpler `where` on the list;

```dart
packages = packages.where((package) => package.isFlutterPackage == hasFlutter).toList();
```

(untested quickly typed code 😅 )

Reply:
Even better!
Implemented and tested my usecases :ok_hand: